README.md

```diff
@@ -66,21 +66,27 @@ The minimum amount of voting power require to pass a proposal of each type it's
 
 ## Database setup
 
-Make sure you have Postgres running.
+Make sure you have Postgres running, or install it and start it up:
+
+```shell
+brew install postgresql@14  
+brew services start postgresql@14
+createdb
+```
 
 You can create a DB from scratch, or you can use the `import-db.sh` script to create a new DB using the `development.sql` dump.
 
 ### Importing DB dump
 
 ```bash
-bash ./scripts/import-db.sh USERNAME [DATABASE_NAME] [DUMP_FILE]
+bash import-db.sh USERNAME [DATABASE_NAME] [DUMP_FILE]
```

Comment:
Isn't this file in `./scripts/import-db.sh`?

Reply:
Ok, it wasn't working like that because of the `.`.
I'll leave it as `bash /scripts/import-db.sh USERNAME [DATABASE_NAME] [DUMP_FILE]`